helpers/download-file.js

```diff
@@ -68,6 +73,7 @@ const downloadFile = settings => function(fileType) {
           case 'pdf':
             return parsePDF(data).then(pdf => pdf.text.replace(/\s/g, ' '));
           case 'word':
+          case 'csv':
             return data.toString('utf8');
```

Comment:
Depending on what you want to do, it might be worth putting these through a csv parser to get actual structured data out.

Reply:
Also, the way this helper selects links generally is a bit of a shit-show.

I might advocate for adding a custom handler onto an element so we can do something like 

```js
const data = browser.$('a=Download CSV file').download('csv');
```

Reply:
```
const subscribersCsv = browser.downloadFile({ type: 'csv', selector: 'a*=Operational newsletter' });
holcs.forEach(email => assert.ok(subscribersCsv.includes(email)));
```

Is how I'm using it - for this test I only care that the csv includes the email address, not what column it's in.